src/components/Button.re

```diff
@@ -25,7 +25,10 @@ module Styles = {
       getBackgroundColor(~type_);
 
     let rules = [
-      padding2(~v=rem(1.1), ~h=rem(1.3)),
+      padding2(
+        ~v=px(Theme.Spacing.baseHalf),
+        ~h=px(Theme.Spacing.baseHalf),
```

Comment:
since both values here are equal, we can just use `padding` instead of `padding2`

Reply:
:man_facepalming: indeed! thank you